build/common.props

```diff
@@ -11,7 +11,13 @@
     <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
   </PropertyGroup>
 
-   <ItemGroup>
+   <!-- Workaround for RC3 VS -->
+  <ItemGroup Condition="'$(BuildingInsideVisualStudio)'!='true'">
```

Comment:
It seemed to be the only way to not get VS to complain. I think I have the same build installed - `26118.1.d15rel`. In theory it would be the same amount of work to remove this node once we have an updated CLI.